src/plugins/vis_type_timeseries/public/application/components/panel_config/timeseries.tsx

```diff
@@ -227,6 +228,21 @@ export class TimeseriesPanelConfig extends Component<
                   />
                 </EuiFormRow>
               </EuiFlexItem>
+              <EuiFlexItem grow={false}>
+                <EuiFormLabel>
```

Comment:
Please use EuiFormRow component like we already did for all similar places. See #97796

Reply:
Added EuiFormRow